include/mujoco_ros2_simulation/mujoco_system_interface.hpp

```diff
@@ -135,6 +135,14 @@ class MujocoSystemInterface : public hardware_interface::SystemInterface
    */
   void register_sensors(const hardware_interface::HardwareInfo& info);
 
+  /**
+   * @brief Sets the initial positions and control values from provided filepath
```

Comment:
It's everything, right?

```suggestion
   * @brief Sets the initial simulation conditions (pos, vel, ctrl) values from provided filepath.
```